regression/pages/lms/dashboard_lms.py

```diff
@@ -29,3 +29,13 @@ def click_donate_button(self):
         Clicks donate button on Dashboard
         """
         self.q(css='.action-donate').click()
+
+    def logout_lms(self):
```

Comment:
nit: I'd call this `click_logout_lms` because you're doing the `wait` separately (in fact, one of the points of your test.)